### PR TITLE
Adding simple setup/run script that eases usage

### PR DESCRIPTION
**Pull request by Ocramius**

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+scriptDir=$(pwd)
+
+cd "$scriptDir"/vagrant
+
+vagrant up
+vagrant ssh -c "cd share/docker && sh run.sh"
```
Comment (cordoval) [inline run.sh:8]: lets not use vagrant please, that is old, let's use docker for this

Comment (Ocramius) [inline run.sh:8]: Docker + macos = a mess.

Vagrant all the way for now, as I've already discussed before.

Comment (cordoval) [inline run.sh:8]: i never said macos, neo welcome to the real world https://github.com/timglabisch/dockergo

Comment (Ocramius) [inline run.sh:8]: I want something people can use ootb, regardless if it's slow or fast.

The first iteration aims at just building something usable. I'm not going to throw in crazy experiments for the sake of it ;-)

**Pull request closed without merging**